examples/medplum-hello-world/src/components/PatientOverview.tsx

```diff
@@ -12,63 +12,13 @@ import { formatDate } from '@medplum/core';
  *   - Patient/Provider profile photo
  * */
 export function PatientOverview(): JSX.Element {
-  const { patient, orders, reports } = usePatientInfo().data;
+  const { id } = useParams();
   return (
     <Document>
-      <h3>Patient Overview</h3>
-
-      <div className="patient-sidebar">
-        <div className="patient-title">
-          <ResourceName value={patient} />
-        </div>
-        <h3>Birth Date</h3>
-        <div>{patient.birthDate}</div>
-        <h3>Address</h3>
-        {patient.address?.map((a, i) => (
-          <div key={`address-${i}`}>
-            <AddressDisplay value={a} />
-          </div>
-        ))}
-        <h3>Contact</h3>
-        {patient.telecom?.map((t, i) => (
-          <div key={`contact-${i}`}>
-            <ContactPointDisplay value={t} />
-          </div>
-        ))}
-      </div>
-      <div className="patient-demographics">
-        <h3>Demographics</h3>
-        <div>Created Date: {formatDate(patient.meta?.lastUpdated)}</div>
-        <table>
-          <tbody>
-            <tr>
-              <td>Prefix: {patient?.name?.[0]?.prefix}</td>
-              <td>First: {patient?.name?.[0]?.given?.[0]}</td>
-              <td>Middle: {patient?.name?.[0]?.given?.[1]}</td>
-              <td>Last: {patient?.name?.[0]?.family}</td>
-              <td>Suffix: {patient?.name?.[0]?.suffix}</td>
-            </tr>
-          </tbody>
-        </table>
-        <h3>Orders ({orders?.length})</h3>
-        <ul>
-          {orders?.map((o, i) => (
-            <li key={`order-${i}`}>
-              <a href={`/ServiceRequest/${o.id}`}>{o.code?.text}</a> ({formatDate(o.meta?.lastUpdated)})
-            </li>
-          ))}
-        </ul>
-        <h3>Reports ({reports?.length})</h3>
-        <ul>
-          {reports?.map((o, i) => (
-            <li key={`report-${i}`}>
-              <a href={`/DiagnosticReport/${o.id}`}>
-                {o.code?.text} ({formatDate(o.meta?.lastUpdated)})
-              </a>
-            </li>
-          ))}
-        </ul>
-      </div>
+      <Title order={3} mb="xl">
+        Patient Overview
+      </Title>
+      <ResourceTable value={{ reference: `Patient/${id}` }} />
```

Comment:
This might be controversial, but I think we should avoid GraphQL for the simple "Hello World" demo.

The `<ResourceTable>` component is a quick and easy way to show a resource.

Like many Medplum components, `<ResourceTable>` can accept either a `Resource` or a `Reference`.  The advantage of passing in a `Reference` is that you don't need to preload the resource.  So, in this case, we can just pass in the `id`.